rajawali/src/main/java/org/rajawali3d/Object3D.java

```diff
@@ -48,6 +48,8 @@
     public static final int BLUE = 2;
     public static final int ALPHA = 3;
 
+	public static final int UNPICKABLE = Color.argb(255, 255, 255, 255);
```

Comment:
Since color picking doesn't have much meaning with transparent objects, perhaps it would be better to use a 0 alpha channel, allowing us to use white as a picking color.